Variable may be used uninitialized

### LVGL version

9.2.2

### Platform

Wayland on Fedora 43 Linux

### What happened?

Code does not compile

```
In function â€˜get_from_poolâ€™,
    inlined from â€˜smm_acquireâ€™ at /builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/drivers/wayland/lv_wayland_smm.c:249:15:
/builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/drivers/wayland/lv_wayland_smm.c:454:24: error: â€˜buf_szâ€™ may be used uninitialized [-Werror=maybe-uninitialized]
  454 |                 buf_sz = (grp->size - buf_sz);
      |                 ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
/builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/drivers/wayland/lv_wayland_smm.c: In function â€˜smm_acquireâ€™:
/builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/drivers/wayland/lv_wayland_smm.c:409:12: note: â€˜buf_szâ€™ was declared here
  409 |     size_t buf_sz;
      |            ^~~~~~
cc1: all warnings being treated as errors
```
https://download.copr.fedorainfracloud.org/results/fed500/lvgl/fedora-rawhide-x86_64/08812651-lvgl/builder-live.log.gz

### How to reproduce?

_No response_

--- a/src/drivers/wayland/lv_wayland_smm.c
+++ b/src/drivers/wayland/lv_wayland_smm.c
@@ -406,7 +406,7 @@ size_t calc_buffer_size(struct smm_buffer * buf)
 struct smm_buffer * get_from_pool(struct smm_group * grp)
 {
     int ret;
-    size_t buf_sz;
+    size_t buf_sz = 0;
     struct smm_buffer * buf;
     struct smm_buffer * last = NULL;
 

